### PR TITLE
Add slis

### PR DESCRIPTION
**Pull request by anishasthana**
## Related Issues and Dependencies

…

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

This PR will add information on all the metrics we are currently monitoring, as well as their current statuses. I'd recommend looking at this PR by commit.

### CODE REVIEW

File: RUNBOOK.md [hunk 1]
```diff
@@ -5,6 +5,51 @@
...
+| Number of Source Repos    | count(count by (repo) (argocd_app_info{project=~"$Project"}))        | Done     |
+| Rate of Change            | To Be Implemented                                                    | Not Done |
+
+### Service Level Indicators for App Owners
+
+These metrics are possible SLIs for the App Owners
+
+| SLI               | Query                                                                                                          | Status |
+|-------------------|----------------------------------------------------------------------------------------------------------------|--------|
+| App Sync Failures | ceil(increase(argocd_app_sync_total{dest_server=~"\$Server", project=\~"$Project", phase=~"Error\|Failed"}[10m])) | Done   |
```
Comment (accorvin) [inline RUNBOOK.md:41]: This is being rendered in GitHub with strikethroughs in some words.

File: RUNBOOK.md [hunk 2]
```diff
@@ -5,6 +5,51 @@
...
+
+### Possible Service Level Indicators
+
+These metrics are interesting to look at but their usefulness has not yet been determined.
+
+| Metric                             | Query                                                                                                                                                                                                                 | Possible Usage | Status   |
+|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|----------|
+| Percentage of Apps in Sync         | sum(argocd_app_info{sync_status="Synced", dest_server=~"\$Server", dest_namespace=~"\$Namespace", project=\~"\$Project"})/sum(argocd_app_info{dest_server=~"\$Server", dest_namespace=~"\$Namespace", project=~"\$Project"}) | App Owner SLI  | Done     |
```
Comment (accorvin) [inline RUNBOOK.md:49]: This is being rendered in GitHub with strikethroughs in some words.

File: RUNBOOK.md [hunk 3]
```diff
@@ -5,6 +5,51 @@
...
 
+## Things to look at
+
+We are tracking a number of metrics at our operations dashboard. What follows are the distinct categories of metrics
+
+### Service Level Indicators
+
+SLIs are carefully defined quantitative measures of some aspect of the level of service that is provided. You can find more information on SLIs in the [Site Reliability Engineering Book](https://landing.google.com/sre/sre-book/chapters/service-level-objectives).
+
+| SLI                                    | Query                                                                                                     | Status                        |
```
Comment (accorvin) [inline RUNBOOK.md:16]: I think it might be useful to have a short description for the non-obvious SLIs in this list

File: RUNBOOK.md (hunk 1 above)
Comment (anishasthana) [reply RUNBOOK.md]: Weird, my local preview didn't have those.. Fixed now.

File: RUNBOOK.md (hunk 2 above)
Comment (anishasthana) [reply RUNBOOK.md]: Weird, my local preview didn't have those.. Fixed now.

File: RUNBOOK.md (hunk 3 above)
Comment (accorvin) [reply RUNBOOK.md]: I think this still might be necessary. @goern what do you think? You seemed to be particularly interested in understanding the meaning of various metrics that we measure for ArgoCD. Does the documentation in this PR answer your questions?

Comment (anishasthana) [reply RUNBOOK.md]: (it's not complete yet) I'm working on updating the documentation for some of the SLIs

Comment (anishasthana) [reply RUNBOOK.md]: Added now.

**Comment by HumairAK** [comment]
lgtm, I say :ship:

**Comment by anishasthana** [comment]
@accorvin could you take another look?

**Comment by accorvin** [comment]
🚢 This looks really good to me.

**Pull request merged**